engines/default.lua

```diff
@@ -204,20 +204,29 @@ minetest.register_node("jumpdrive:engine", {
 	end,
 
 	-- inventory protection
+	allow_metadata_inventory_move = function(pos, _, _, _, _, count, player)
+		if (not player) -- no player given
+			or (not player:is_player()) -- not a player
+			or minetest.is_protected(pos, player:get_player_name()) -- protected
+		then return 0 end
```

Comment:
Parentheses and comments are not necessary.
 ```suggestion
		if not player or not player:is_player() or minetest.is_protected(pos, player:get_player_name()) then
			return 0
		end
```
Not sure if `player:is_player()` check is really needed either 🤔

Reply:
yes, I'll remove the comments. The parenthis are not needed but they help ensure that the reader understands what is going on in chained and/or checks. I consider it a maintenance speed up.

I think the is_player check is ok. Some server may have a 'thief' mob installed that tries to take stuff from inventories.
Or similar.